integration-test/rules/multiple-queries.yml

```diff
@@ -0,0 +1,17 @@
+title: Multiple Queries Test Rule
+id: 26654045-413c-4db1-be5f-36f34733a865
+status: test
+description: Tests that multiple queries are combined correctly.
+author: kleimkuhler
+date: 2025-11-06
+logsource:
+  category: integration-test
+detection:
+  selection1:
+    username: user1
+  selection2:
+    username: user2
+  condition: selection1 or selection2
```

Comment:
This will only generate a single query - to generate multiple queries, conditions needs to be a list of conditions (which is valid, but a rarely used Sigma feature). Try:
```
condition:
  - selection1
  - selection2
```